sources/assets/Xenko.Core.Assets.CompilerClient/Xenko.Core.Assets.CompilerClient.csproj

```diff
@@ -27,11 +27,15 @@
       <Link>Package.Constants.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build" Version="15.7.179" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.Build.Locator" Version="1.0.18" ExcludeAssets="runtime" />
```

Comment:
I don't think the locator should be excluded at runtime. Or it means it is not needed here.